docs/dev/guide/hello-world.md

```diff
@@ -493,14 +499,14 @@ import { ethers } from "ethers";
 ```javascript
 async getBalance() {
     // Getting the balance for the signer in the selected token
-    const balanceInUnits = await this.signer.getBalance(this.selectedToken.address);
+    const balanceInUnits = await this.signer.getBalance(this.selectedToken.l2Address);
     // To display the number of tokens in the human-readable format, we need to format them,
     // e.g. if balanceInUnits returns 500000000000000000 wei of ETH, we want to display 0.5 ETH the user
     return ethers.utils.formatUnits(balanceInUnits, this.selectedToken.decimals);
 },
 ```
 
-3. stimate the fee:
+3.  Estimate the fee:
```

Comment:
Looks like you accidentally added an extra space here, instead of my suggestion above